models/model.js

```diff
@@ -9,16 +9,9 @@ exports.selectAllTopics = (req, res)=>{
    })
 }
 
-exports.selectArticleId = (article_id) =>{
-   return db.query('SELECT * FROM articles WHERE article_id = $1', [article_id])
-   .then(({ rows })=>{
-      
-      if(rows.length !== 0){
-         return rows[0]
-     }
-     else{
-         return Promise.reject({status: 404, msg: "Not Found"})
-     }
+exports.selectAllArticles = (req, res) =>{
+   return db.query('SELECT article_id, title, topic, author, created_at, votes, article_img_url FROM articles ORDER BY created_at DESC;')
+   .then(({rows})=>{
+    return rows
```

Comment:
To pass the comment_count part of the test, you will need to use an aggregate function. The is a similar problem in the notes here:
https://notes.northcoders.com/courses/js-back-end/sql-relationships#group-by-and-aggregate-functions